src/GeneratedHydrator/Configuration.php

```diff
@@ -220,4 +225,20 @@ public function getClassNameInflector()
 
         return $this->classNameInflector;
     }
+
+    /**
+     * @param \PhpParser\NodeVisitor[] $visitors
+     */
+    public function setCustomVisitors(array $customVisitors)
```

Comment:
I added an interface check to the custom visitor setter. I didn't want to use splatting because it would change the setter signature. You'd expect an array with a setter name like `setCustomVisitors`. It also means that what you get back from the getter is what you can pass into the setter.